### PR TITLE
Update readmes to match common design (Group Layers & Hydrography categories) 

### PR DESCRIPTION
**Pull request by JonLavi**

### CODE REVIEW

File: group_layers/group-layers/README.md [hunk 1]
```diff
@@ -2,22 +2,27 @@
...
 
-The layers in the map will be displayed in a table of contents. Toggle the checkbox next to a layer's name to change its visibility.
+The layers in the map will be displayed in a table of contents. Toggle the checkbox next to a layer's name to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
 
 ## How it works
 
 1. Create an empty `GroupLayer`.
 2. Add a child layer to the group layer's layers collection.
-3. To toggle the visibility of the group, simply change the group layer's visibility property.
+3. To build the table of contents:
```
Comment (Rachael-E) [inline group_layers/group-layers/README.md]: we do this quite differently in Java and the only step we follow here is checking if `isShowChildrenInLegend` is true. I'm not sure how best to write up our steps when a lot of it is Java logic to recursively populate the table of contents with layer names?

File: group_layers/group-layers/README.md [hunk 2]
```diff
@@ -27,12 +32,8 @@ The layers in the map will be displayed in a table of contents. Toggle the check
 
 The full extent of a group layer may change when child layers are added/removed. Group layers do not have a spatial reference, but the full extent will have the spatial reference of the first child layer.
 
-Group layers can be saved to web scenes. In web maps, group layers will be flattened in the web map's operational layers.
-
-The implementation shown here makes use of a custom tree cell for displaying layers in a tree view. In the custom
```
Comment (Rachael-E) [inline group_layers/group-layers/README.md:32]: With the above comment in mind, we should keep this information in here for the Java sample.

**Review by Rachael-E — CHANGES_REQUESTED** [review]
with this one we just need to move back in some of the removed Java specific instruction, thanks @JonLavi

File: group_layers/group-layers/README.md (hunk 1 above)
Comment (JonLavi) [reply group_layers/group-layers/README.md]: Thanks for pointing this out. Considering most of the building of the 'tree' is UI focused, I've reverted back to the old java 'how it works' section, with a few additions. Looks like the main point is to show how to use the `GroupLayer` class and toggling visibility, so we don't need to do much more than that. What do you think?

File: group_layers/group-layers/README.md (hunk 2 above)
Comment (JonLavi) [reply group_layers/group-layers/README.md:32]: Yep, I've added this back in. Thanks!

**Comment by JonLavi** [comment]
Thanks @Rachael-E , could you have a second look at the updated sections please?

**Review by Rachael-E — APPROVED** [review]
cool thanks @JonLavi , and yup, I agree with just showing the main API call in the how it works section.

**Pull request merged**